README.md

```diff
@@ -88,6 +88,16 @@ Under the run/ debug configuration drop down menu, click `Edit Configurations...
 
 See [PushNotifications.md](PushNotifications.md) for detailed information on getting PN working with the example app.
 
+###### **Please note:**
```

Comment:
I think this needs to be H4, given previous heading is H3... or, in other words, this feels like misuse of a heading so I would perhaps also change the heading text (also makes this deep linkable):

```suggestion
#### Important Note regarding Firebase and Push Notifications
```